addon/behaviors/row-expansion.js

```diff
@@ -0,0 +1,23 @@
+import Behavior from 'ember-light-table/behaviors/behavior';
+
+export default Behavior.extend({
+
+  exclusionGroup: 'row-expansion',
+
+  // passed in
+  multiRow: true,
+
+  init() {
+    this._super.call(this, ...arguments);
+    this.events.onToggleExpandedRow = [ 'rowClick:_all' ];
+  },
+
+  onToggleExpandedRow(ltBody, row) {
```

Comment:
I don't like it either. It was simply to make `mixins/has-behaviors` more generic, i.e. it is simply passing `this` as the first parameter. Not a very good reason, I aggree. Maybe we can instruct `mixins/has-behaviors` what to pass as the first argument. What do you think ?